gpkit/keydict.py

```diff
@@ -45,6 +45,7 @@ def update(self, *args, **kwargs):
         "Iterates through the dictionary created by args and kwargs"
         for k, v in dict(*args, **kwargs).items():
             if hasattr(v, "copy"):
+                # we don't want just a reference (for e.g. numpy arrays)
```

Comment:
Was asking something a little more subtle here. If a user writes their own class (e.g. `pint.Quantity`, but possibly something else, what are the requirements for it to work as a value in a KeyDict?

 - Does it need to be immutable?
 - Does it need to supply a `copy()` method?